runtime/src/bank.rs

```diff
@@ -4229,19 +4232,19 @@ impl Bank {
             process_message_time.as_us()
         );
 
+        let accounts_data_len_delta = process_result
+            .as_ref()
+            .map(|info| info.accounts_data_len_delta)
+            .unwrap_or(0);
```

Comment:
This needs to be after `verify_transaction_account_state_changes` because if that fails, there are no data len changes

Reply:
I was under the impression that `status` will be `Err` if `verify_transaction_account_state_changes()` fails. In commit, I only sum the accounts data len deltas for `Ok` status. Let me know if I've misunderstood something.

I would prefer to have a clearer way to handle this, which I talk about here: https://github.com/solana-labs/solana/pull/25256#discussion_r874128680. Maybe (1) is doable?

Reply:
You are right but status isn't determined until after that rent check. And as noted in my other comment, it looks like you are summing deltas for failed but executed transactions

Reply:
Gotcha, I see now. Thanks, will fix.

Reply:
Done!